codebuild/python2.7.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+env:
+  variables:
+    TOXENV: "py27-integ-slow"
```

Comment:
Nice, this is actually a bit better coverage than before when we couldn't run certain tests in Travis because they caused crashes (https://github.com/aws/aws-dynamodb-encryption-python/pull/81). Looks like CodeBuild handles it all just fine.